### PR TITLE
add get_service_mrg_facts

### PR DESCRIPTION
**Pull request by le9i0nx**
It adds a variable that holds the name of the system initialization

### CODE REVIEW

**Comment by jimi-c** [comment]
Hi @le9i0nx, could you change the name to `service_manager`? I think that having the long form is better to make it obvious what it is. Thanks!

**Comment by jimi-c** [comment]
Also, the `with open` syntax is not python 2.4 compliant (which facts code needs to be), so that will need to be fixed.

File: lib/ansible/module_utils/facts.py [hunk 1]
```diff
@@ -515,6 +521,15 @@ def get_pkg_mgr_facts(self):
         if self.facts['system'] == 'OpenBSD':
                 self.facts['pkg_mgr'] = 'openbsd_pkg'
 
+    def get_service_mrg_facts(self):
+        self.facts['service_mgr'] = 'auto'
```
Comment (bcoca) [inline lib/ansible/module_utils/facts.py]: 'auto' is not a valid value to return from this fact

File: lib/ansible/module_utils/facts.py [hunk 2]
```diff
@@ -515,6 +521,15 @@ def get_pkg_mgr_facts(self):
...
+        for service in Facts.INIT_SYSTEM_MGRS:
```
Comment (bcoca) [inline lib/ansible/module_utils/facts.py]: this is not a good way to detect initd systems, upstart, openrc, rc systems will all be confused into init

**Comment by le9i0nx** [comment]
decided

**Pull request closed without merging**